src/BenchmarksDriver/Program.cs

```diff
@@ -0,0 +1,319 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Benchmarks.ClientJob;
+using Benchmarks.ServerJob;
+using Microsoft.DotNet.Cli.Utils;
+using Microsoft.Extensions.CommandLineUtils;
+using Newtonsoft.Json;
+
+namespace BenchmarkDriver
+{
+    public class Program
+    {
+        private static readonly HttpClient _httpClient = new HttpClient();
+
+        private static readonly Dictionary<Scenario, ClientJob> _clientJobs =
+            new Dictionary<Scenario, ClientJob>()
+            {
+                { Scenario.Plaintext, new ClientJob() { Connections = 256, Threads = 32, Duration = 10, PipelineDepth = 16} },
```

Comment:
It's easy to add this feature, but I'd rather wait until someone actually wants to use it.  If you do  have a real use for it now, I can add it.

I would still want to keep the defaults hardcoded for each scenario, to make it easier for everyone to use the same parameters unless they explicitly want something different.